posydon/popsyn/population_params_default.ini

```diff
@@ -289,13 +289,13 @@
   # convert BinaryStars into DataFrames after evolution
   use_MPI = False
   # use only for local MPI runs
-  metallicity = [1] # [2., 1., 0.45, 0.2, 0.1, 0.01, 0.001, 0.0001]
+  metallicity = [2., 1., 0.45, 0.2, 0.1, 0.01, 0.001, 0.0001]
   # In units of solar metallicity
 
   # Random Number Generation
   entropy = None
   # `None` uses system entropy (recommended)
-  number_of_binaries = 10
+  number_of_binaries = 1000000
```

Comment:
I'm not sure, whether we should make this our default. I guess, a default user will try it on his local machine and not on a HPC, hence this default might be to computationally expensive. (The same applies to the change on metallicity.)

Reply:
Per discussion, I've changed it to 10 and to 1 metallicity. Also adapted in the tutorial.